pkg/plugin/controller/printers.go

```diff
@@ -130,11 +130,94 @@ func ResourcePrinter(request *service.PrintRequest) (plugin.PrintResponse, error
 	return plugin.PrintResponse{
 		Status: vulnerabilities.NewSummarySections(summary),
 		Config: configaudit.NewSummarySections(configAuditSummary),
-		Items: []component.FlexLayoutItem{
-			{
-				Width: component.WidthFull,
-				View:  configaudit.NewReport(workload, configAuditReportsDefined, configAuditReport),
+	}, nil
+}
+
+func ResourceReportTabPrinter(request *service.PrintRequest) (plugin.TabResponse, error) {
+	if request.Object == nil {
+		return plugin.TabResponse{}, errors.New("object is nil")
+	}
+
+	workload, err := getWorkloadFromObject(request.Object)
```

Comment:
We generate ConfigAuditReports for a subset of K8s resources (GVKs). For example, we'll never generate it for Nodes because for Nodes we generate CISKubeBenchReports.
That said, I was wondering if we can we skip adding a Tab for certain GVKs?

Reply:
Checking if the object type is a node and returning an empty tab response is one way, but I don't like having that error message.

Going to patch upstream to skip empty tab responses without errors.

https://github.com/vmware-tanzu/octant/blob/5a8648921cc2779eb62a0ac11147f12aa29f831c/pkg/plugin/grpc.go#L557-L559

Reply:
An empty tab response sounds good. We can improve error handling by upgrading Octant whenever a new version is available.